test/fixtures/mson/enum-variants.json

```diff
@@ -386,6 +427,10 @@
                               {
                                 "element": "string",
                                 "content": "N"
+                              },
+                              {
+                                "element": "string",
+                                "content": "L"
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?